tests/test_loader.py

```diff
@@ -201,3 +202,80 @@ def test_includes_has_invalid_tag():
 
     msg = "Error parsing RAML:"
     assert msg in e.value.args[0]
+
+
+def test_json_ref_in_schema_relative():
+    raml_file = os.path.join(EXAMPLES, "json_include_with_ref.raml")
+    with open(raml_file) as f:
+        raml = loader.RAMLLoader().load(f)
+        expected_data = {
+            "title": "Sample API Demo - JSON Includes",
+            "version": "v1",
+            "baseUri": "http://foo-json.bar",
+            "schemas": [{
+                "json": {
+                    "name": "foo",
+                    "second_name": "bar",
+                    "false": True
+                },
+            }],
+            "/foo": {
+                "displayName": "foo resource"
+            }
+        }
+        assert dict_equal(raml, expected_data)
+
+
+def test_json_ref_in_schema_absolute():
+    # Set up the file with an absolute path
+    with open(os.path.join(EXAMPLES, "includes",
```

Comment:
Take a look at `pytest`'s tmpdir/tmpfile [here](https://pytest.org/latest/tmpdir.html).  Rather than creating a file, leaving it on the file system, and adding it to `.gitignore`, I'd prefer this way if you need to create a JSON file for this purpose.